packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -1076,8 +1122,13 @@ public void unlockCaptureOrientation() {
 
   /** Pause the preview from dart. */
   public void pausePreview() throws CameraAccessException {
-    this.pausedPreview = true;
-    this.captureSession.stopRepeating();
+    if (!this.pausedPreview) {
```

Comment:
How is this change related to the PR?

Reply:
This is actually bug fix (not related to media settings). Removed from this PR. Should i propose bugfix in standalone PR?

Reply:
Yes please!